### ISSUE TITLE
Strange with `remark-lint-list-item-bullet-indent`

### ISSUE THREAD

**Issue by evilebottnawi** [wooorm/remark-lint#132]
I am prefer 4 spaces for indent. But if i use 
```markdown
- foo

- bar

- foo-bar

    - foo-foo
```

I am get a error:

```shell
warning  Incorrect indentation before bullet: remove 2 spaces  list-item-bullet-indent  remark-lint
```
It is possible use 4 spaces for nested list item?

**Comment by wooorm** [comment]
Sure, that’s possible, but not with this rule. I think the [example](https://github.com/wooorm/remark-lint/tree/master/packages/remark-lint-list-item-bullet-indent#example) explains what’s going on.

If you want to use four spaces, in markdown, you should indent the first like too: `foo`, `bar`, and `foo-bar` should have three instead of one space before them!

...or turn this rule off :)

**Comment by evilebottnawi** [comment]
@wooorm Is it not possible to add an option to control this? :smile:  I can do PR.

**Comment by wooorm** [comment]
Well, the problem here is that, for the parser, if you do `- foo`, the content of the list-item starts at the first letter (column: 3) for every line.

The most “standard” way to align list-items is to use one of the following:

`a.md`:

```markdown
- foo

- bar

- foo-bar

  - foo-foo
```

...or, `b.md`:

```markdown
-   foo

-   bar

-   foo-bar

    - foo-foo
```

I think, if you want different alignment, this should be in an external rule instead!

**Comment by evilebottnawi** [comment]
@wooorm thanks for clarification :smile:
